src/core/core.js

```diff
@@ -329,14 +329,14 @@ export default class Core {
       }
     }
 
-    this.storage.set(StorageKeys.DIRECT_ANSWER, {});
     const universalResults = this.storage.get(StorageKeys.UNIVERSAL_RESULTS);
     if (!universalResults || universalResults.searchState !== SearchStates.SEARCH_LOADING) {
       this.storage.set(StorageKeys.UNIVERSAL_RESULTS, UniversalResults.searchLoading());
     }
     this.storage.set(StorageKeys.QUESTION_SUBMISSION, {});
     this.storage.set(StorageKeys.SPELL_CHECK, {});
     this.storage.set(StorageKeys.LOCATION_BIAS, LocationBias.searchLoading());
+    this.storage.set(StorageKeys.DIRECT_ANSWER, DirectAnswer.searchLoading());
```

Comment:
do we need to change the order here? I think it's probably fine in this case, but we try to avoid changing this because messing with the ordering can change behavior in unexpected ways

Reply:
ah no it's not needed. I don't think it would change any behavior, just want to organize it with other `set` call. I will keep it in mind!

Reply:
okay let's move it back then if it's not needed then!

Reply:
changing the order will affect the order that components are rendered, which could break some weird thing consulting is doing